lib/NativeUtils.js

```diff
@@ -11,7 +11,6 @@ module.exports = class NativeUtils extends Utils {
         // TODO: add const variables in spec file and add path spec, remove "tbd"
         android: 'file:///android_asset/www/index.html',
         ios: `${this.context.getUrl()}`,
-        electron: `${this.context.getUrl()}`,
```

Comment:
probably add a comment why `electron` is missing here?

Reply:
// electron is part of ./BrowserUtils.js